src/webui/components/common/Spinner.tsx

```diff
@@ -1,11 +1,15 @@
 
 import { Panel, Row, Spinner } from '^/webui/designsystem';
 
-export default function Component()
+type Props = {
+    readonly active?: boolean;
+};
+
+export default function Component({ active = true }: Props)
```

Comment:
```suggestion
export default function Component({ active }: Props)
```

Do we want to override the default behavior of the design system component?